src/components/FurnitureField/FurnitureField.jsx

```diff
@@ -16,34 +16,41 @@ function FurnitureField({ index, register, removeFurniture }) {
     'Patio Furniture',
   ];
 
+  let deleteButton;
+  if (removeFurniture == null) {
+    deleteButton = null;
+  } else {
+    deleteButton = <Button onClick={() => removeFurniture(index)}>Delete</Button>;
+  }
+
   return (
     <div className={styles['field-section']}>
       <h1 className={styles.title}> Furniture {index + 1} </h1>
       <div className={styles['field-spacing']}>
         <FormLabel>Items to be Donated</FormLabel>
-        <Select defaultChecked="Dressers" {...register(`furnitureField.${index}.itemName`)}>
+        <Select
+          placeholder="Select furniture"
+          defaultChecked="Select furniture"
+          {...register(`furnitureField.${index}.itemName`)}
+        >
           {furnitureOptions.map((furnitureItem, i) => (
             // eslint-disable-next-line react/no-array-index-key
             <option key={i}>{furnitureItem}</option>
           ))}
         </Select>
       </div>
-      <div className={styles['field-spacing']}>
-        <FormLabel>Furniture Image Link</FormLabel>
-        <Input defaultValue="" {...register(`furnitureField.${index}.imageLink`)} />
-      </div>
-      <div className={styles['field-spacing']}>
-        <FormLabel>Description</FormLabel>
-        <Input defaultValue="" {...register(`furnitureField.${index}.description`)} />
-      </div>
-      <Button onClick={() => removeFurniture(index)}>Delete</Button>
+      {/* <Button onClick={() => removeFurniture(index)}>Delete</Button> */}
+      {deleteButton}
```

Comment:
Instead of using a basic `if`/`else` block to show the delete button or not, you can just use the `&&` operator, which uses short circuiting to conditionally display the component. See [here](https://dev.to/harlessmark/short-circuit-evaluation-with-react-3dn4) for an example.